default.nix

```diff
@@ -69,12 +69,6 @@ stdenv.mkDerivation rec {
     ++ [ dune_2 ] # Maybe the next build system
   );
 
-  # Since #12604, ocamlfind looks for num when building plugins
-  # This follows a similar change in the nixpkgs repo (cf. NixOS/nixpkgs#94230)
-  propagatedBuildInputs = [
-    ocamlPackages.num
```

Comment:
This removal is wrong. Please rebase as soon as #13015 is merged: num should go but zarith should stay as a propagated-build-input.

Reply:
Done, thanks. Indeed I had seen #13015 but you are right I should have left the empty list.

Reply:
Thanks. Apologies for the unfortunate timing.